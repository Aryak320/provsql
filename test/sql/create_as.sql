\set ECHO none
SET search_path TO public,provsql;

CREATE TABLE x (n integer);
INSERT INTO X VALUES (10),(20);
SELECT add_provenance('x');

CREATE TABLE u AS (SELECT * FROM x);

\d u

DROP TABLE x;
DROP TABLE u;
