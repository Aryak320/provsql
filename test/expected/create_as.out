\set ECHO none
 add_provenance 
----------------
 
(1 row)

 attname |     typname      
---------+------------------
 n       | int4
 provsql | provenance_token
(2 rows)

